### PR TITLE
🍒 Add CV 3 metadata

### PR DESCRIPTION
**Pull request by adrianpetre**

### CODE REVIEW

**Review by vivekjhauipath — APPROVED** [review]
LGTM

File: metadata/ComputerVision__3__metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "name": "ComputerVision",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10",
```
Comment (vnaren23) [inline metadata/ComputerVision__3__metadata.json:20]: Shouldn't this be 22.4?

**Review by vnaren23 — APPROVED** [review]
LGTM

File: metadata/ComputerVision__3__metadata.json (hunk 1 above)
Comment (adrianpetre) [reply metadata/ComputerVision__3__metadata.json:20]: The model should work with the 21.10 AIC as well, as per this conversation https://uipath-product.slack.com/archives/CN7P410DD/p1646142714147669

**Pull request merged**